src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { chmodSync, constants, exists, unlinkSync } from 'fs';
+import { homedir } from 'os';
+import { join } from 'path';
+import { readFile, writeFile } from 'jsonfile';
+
+// tslint:disable-next-line:no-any
+type jsonAny = any;
```

Comment:
I think we can just use `object` instead of this type.
Delete this.

Reply:
With unknown json object, `any` is usually the best option... Unless I can add generics into the definition.